cypher-editor-support/src/completion/AutoCompletion.js

```diff
@@ -203,7 +211,10 @@ export class AutoCompletion {
       .map(t => t.complete(types, query))
       .reduce((acc, items) => [...acc, ...items], [])
       .filter(completionItemFilter);
-
+    if (text[0] === '$') {
```

Comment:
I am not sure that we should delete dollar sign here.

Maybe we need to add dollar sign to schema?

OR

Extract parameter name from an element.